lib/ae_mdw_web/controllers/aex9_controller.ex

```diff
@@ -204,7 +204,7 @@ defmodule AeMdwWeb.Aex9Controller do
   end
 
   def balance_for_hash_reply(conn, contract_pk, account_pk, {type, block_hash, height}) do
-    {amount, _} = DBN.aex9_balance(contract_pk, account_pk, {height, block_hash})
+    {amount, _} = DBN.aex9_balance(contract_pk, account_pk, {type, height, block_hash})
```

Comment:
I would rather avoid operational changes in this PR. If it's a bug, better report it to keep the tracking also.
`_type` would suffice for now.

Reply:
Removed these changes from the PR